docs/sandbox-run.md

```
@@ -245,7 +245,10 @@ exec harnx-sandbox-run \
   --extra-rwx '$GIT_COMMON_DIR' \
   -- gemini --yolo "$@"
 ```
-If the agent needs access to additional directories, grant them with `--extra-rwx /path/to/dir` in the shim script.
+If the agent needs access to additional directories, grant them with `--extra-rwx /path/to/dir` in the shim script to add them
+permanently, or use environment variables (see below for the list) to add them temporarily:
+
+    HARNX_BASH_EXTRA_READABLE=~/Projects/some-reference-project claude
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)